core/dbt/parser/unit_tests.py

```diff
@@ -214,6 +222,9 @@ def _get_original_input_node(self, input: str, tested_node: ModelNode):
             else:
                 raise InvalidUnitTestGivenInput(input=input)
 
+        if not original_input_node:
+            raise InvalidUnitTestGivenInput(input=input)
```

Comment:
How do we end up here?  Can we test this?

Reply:
I think we'd hit this if a unit test case was specified against a model or source that did not exist in the manifest

Reply:
If somebody has an invalid model ref in the given input, there would be an error.  It's not actually related to this pr, it's base functionality, but I can add a test.

Reply:
Done.